mock/src/test/java/jinsist/mock/testtypes/TestFactories.java

```diff
@@ -23,7 +23,7 @@ protected TestFactories() throws NoSuchMethodException {
     }
 
     protected Arguments argumentsOf(Object... values) {
-        return new Arguments(Arrays.stream(values).map(EqualsMatcher::new).collect(toList()));
+        return new Arguments(stream(values).map(value -> new EqualsMatcher<>(value)).collect(toList()));
```

Comment:
To avoid warning "Type safety: The constructor EqualsMatcher(Object) belongs to the raw type EqualsMatcher. References to generic type EqualsMatcher<T> should be parameterized"